pandapipes/create.py

```diff
@@ -429,13 +430,23 @@ def create_pipe(net, from_junction, to_junction, std_type, length_km, k_mm=0.2,
         >>>             std_type='315_PE_80_SDR_17', length_km=1)
 
     """
+    from pandapipes.toolbox import calculate_alpha
     add_new_component(net, Pipe)
 
     index = _get_index_with_check(net, "pipe", index)
     _check_branch(net, "Pipe", index, from_junction, to_junction)
     _check_std_type(net, std_type, "pipe", "create_pipe")
 
     pipe_parameter = load_std_type(net, std_type, "pipe")
+
+    if pd.notna(pipe_parameter['u_w/mk']) and alpha_w_per_m2k == 0.0:
```

Comment:
I would prefer to use u_w_per_mk here in analogy to the naming of other parameters.

Reply:
when renaming u to alpha in table, this is not needed anymore